composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "profibro/magento2-module-paystack",
+  "name": "pstk/paystack-magento2-module",
```

Comment:
why don't you want to use `paystack` or `paystackhq` as the vendor name?

Reply:
@ibrahimlawal it seems Magento does not allow a vendor-name that has the word 'pay' in it. They kept rejecting paystack or paystackhq or paystackpayments. Hence I went for the abbreviation

Reply:
Oh I see